core/src/main/scala/cats/syntax/applicativeError.scala

```diff
@@ -3,12 +3,18 @@ package syntax
 
 import cats.data.{Xor, XorT}
 
-trait ApplicativeErrorSyntax {
+trait ApplicativeErrorSyntax1 {
+  implicit def applicativeErrorSyntax[F[_], E, A](fa: F[A])(implicit F: ApplicativeError[F, E]): ApplicativeErrorOps[F, E, A] =
+    new ApplicativeErrorOps[F, E, A](fa)
+}
+
+trait ApplicativeErrorSyntax extends ApplicativeErrorSyntax1 {
   implicit def applicativeErrorIdSyntax[E](e: E): ApplicativeErrorIdOps[E] =
     new ApplicativeErrorIdOps(e)
 
-  implicit def applicativeErrorSyntax[F[_, _], E, A](fa: F[E, A])(implicit F: ApplicativeError[F[E, ?], E]): ApplicativeErrorOps[F[E, ?], E, A] =
+  implicit def applicativeErrorSyntax2[F[_, _], E, A](fa: F[E, A])(implicit F: ApplicativeError[F[E, ?], E]): ApplicativeErrorOps[F[E, ?], E, A] =
```

Comment:
@kailuowang yeah, that sounds reasonable to me. Have you by any chance checked that works when using the SI-2712 fix?

@travisbrown do you have any strong opinions on this? I think you originally added the applicative error syntax.